chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
I really really would like to avoid making special cases for submits for every website out there.
Could you let me know what made you go this direction?